packages/library-base/src/commonMain/kotlin/io/realm/internal/RealmLog.kt

```diff
@@ -75,4 +75,22 @@ class RealmLog(val tag: String = "REALM", val configuration: LogConfiguration) {
             }
         }
     }
+
+    override fun equals(other: Any?): Boolean {
```

Comment:
Where are `RealmLog`s compared? If essential, then we might need tests for these

Reply:
Done